src/index.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 'use strict';
+
+const express = require('express');
+const cors = require('cors');
+
+require('dotenv').config();
+
+const cookieParser = require('cookie-parser');
+const { authRouter } = require('./routes/auth.route.js');
+const { userRouter } = require('./routes/user.route.js');
+const { errorMiddleware } = require('./middlewares/errorMiddleware.js');
+
+const PORT = process.env.PORT || 3001;
+
+const app = express();
+
+app.use(
+  cors({
+    origin: process.env.CLIENT_URL,
```

Comment:
Ensure consistent use of the `CLIENT_URL` environment variable across your application to avoid configuration issues.